spec/requests/api/v2/exchange_rates/files_controller_spec.rb

```diff
@@ -54,5 +55,39 @@
         expect(response.body).to eq(data)
       end
     end
+
+    context 'when requesting HMRC CSV format' do
+      let(:type) { 'monthly_csv_hmrc' }
+      let(:format) { 'csv' }
```

Comment:
dont think this is used?

Reply:
we are making use of it in this PR, see the page links